lib/homieDevice.js

```diff
@@ -117,7 +117,7 @@ proto.setup = function(quiet) {
   })
 
   t.mqttClient.on('message', function (topic, message) {
-    t.onMessage(topic, message.toString());
```

Comment:
Why did you remove the toString() here? This will break existing clients because if a buffer comes in, it won't be converted for them. Did you need a feature that this prevented?

Reply:
One testcase failed with a null pointer exception with the toString method. It was only a test and I forgot to add a check